dd-sdk-android/src/main/kotlin/com/datadog/android/DatadogConfig.kt

```diff
@@ -253,20 +255,18 @@ private constructor(
          * tap events and automatically send those as RUM UserActions for you.
          */
         fun trackGestures(): Builder {
-            rumConfig = rumConfig.copy(trackGestures = true)
+            rumConfig =
+                configWithTrackingStrategy(GesturesTrackingStrategy(DatadogGesturesTracker()))
             return this
         }
 
         /**
-         * Sets the automatic view tracking strategy used by the SDK.
-         * By default this is NONE.
-         * @param strategy as the strategy to be used by the automatic view tracker.
-         * @see ViewTrackingStrategy.TRACK_ACTIVITIES_AS_VIEWS
-         * @see ViewTrackingStrategy.TRACK_FRAGMENTS_AS_VIEWS
-         * @see ViewTrackingStrategy.NONE
+         * Provides a view tracking strategy for the SDK.
+         * @param strategy that should be used
+         * (e.g. TrackActivitiesViewsStrategy, TrackFragmentsAsViewsStrategy)
          */
-        fun trackViews(strategy: ViewTrackingStrategy): Builder {
-            rumConfig = rumConfig.copy(viewTrackingStrategy = strategy)
+        fun withViewTrackingStrategy(strategy: TrackingStrategy): Builder {
```

Comment:
I'd create a `setViewTrackingStrategy` instead, and have a single field for view tracking and another one for gestureTracking. 
With what you're currently doiing, one could add both Activity and Fragment tracking which will result in unpredictable behavior.